src/theory/arith/theory_arith_private.cpp

```diff
@@ -138,7 +138,7 @@ TheoryArithPrivate::TheoryArithPrivate(TheoryArith& containing,
                           d_partialModel,
                           RaiseEqualityEngineConflict(*this),
                           d_pnm),
-      d_cmEnabled(c, options::arithCongMan()),
+      d_cmEnabled(c, options().arith.arithCongMan),
```

Comment:
It would probably be cleaner to have the method options getter return `options().arith`, is this possible?